### ISSUE TITLE
Links to models are broken

### ISSUE THREAD

**Issue by minian** [Hvitgar/stylize-datasets#14]
The links in models/download_models.sh, as well as those specified for manual download in the README ([vgg](https://drive.google.com/file/d/108uza-dsmwvbW2zv-G73jtVcMU_2Nb7Y/view)/[decoder](https://drive.google.com/file/d/1w9r1NoYnn7tql1VYG3qDUzkbIks24RBQ/view)) are broken.

**Comment by rajankita** [comment]
Yes, please update the links.

**Comment by rajankita** [comment]
I found this to be helpful: 

The links at https://github.com/rgeirhos/Stylized-ImageNet/blob/master/code/models/download_models.sh are working fine. You can download torch models from there, and then convert to pytorch as follows:

```
cd models
wget -c https://s3.amazonaws.com/xunhuang-public/adain/decoder.t7
wget -c https://s3.amazonaws.com/xunhuang-public/adain/decoder-content-similar.t7
wget -c https://s3.amazonaws.com/xunhuang-public/adain/vgg_normalised.t7
cd .. 

# Convert models to pytorch
python3 torch_to_pytorch.py --model models/vgg_normalised.t7
python3 torch_to_pytorch.py --model models/decoder.t7
```

**Comment by Jeshofm** [comment]
The links in the pytorch-AdaIN repo where changed, but it is fixed now. 
See the downloads in https://github.com/naoto0804/pytorch-AdaIN

**Comment by bmitzkus** [comment]
In general, I'd like to avoid with the torch_to_pytorch.py script (iirc they only work with legacy pytorch versions) - as google drive links are likely subject to changes in the future, it might be a good idea to just reference naoto0804's repository (as we are not hosting the weights ourselves)

**Comment by bmitzkus** [comment]
This is "fixed" in PR #15 - the Readme now just points to https://github.com/naoto0804/pytorch-AdaIN directly, if they don't provide their weights anymore, we won't either, as we didn't train the models ourselves.
